helpers/eslint-config-plasma/package.json

```diff
@@ -15,7 +15,7 @@
         "eslint-plugin-prefer-arrow": "1.2.3",
         "eslint-plugin-react": "7.34.3",
         "eslint-plugin-react-hooks": "4.6.2",
-        "eslint-plugin-unused-imports": "3.2.0"
+        "eslint-plugin-unused-imports": "4.0.0"
```

Comment:
Not sure we can merge this right now

<img width="529" alt="image" src="https://github.com/coveo/plasma/assets/35579930/986d040d-7c06-4171-9ae8-716ab50f9e2d">